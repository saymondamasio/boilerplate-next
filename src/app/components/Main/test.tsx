import { render, screen } from '@testing-library/react'

import { Main } from '.'

describe('<Main />', () => {
	it('should render the heading', () => {
		const { container } = render(<Main title="Hello" />)

		expect(screen.getByRole('heading', { name: /Hello/i })).toBeInTheDocument()

		expect(container.firstChild).toMatchSnapshot()
	})
})
