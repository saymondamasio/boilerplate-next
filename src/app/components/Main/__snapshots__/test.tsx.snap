// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Main /> > should render the heading 1`] = `
<div>
  <h1>
    Hello
  </h1>
</div>
`;
